djstripe/exceptions.py

```diff
@@ -1,14 +1,28 @@
 # -*- coding: utf-8 -*-
+"""
+.. module:: djstripe.exceptions.
+
+   :synopsis: dj-stripe Exceptions.
+
+.. moduleauthor:: N/A
```

Comment:
This one is just me
https://github.com/kavdev/dj-stripe/blame/%23162-api-updates-through_2015-07-28/djstripe/exceptions.py